src/print.js

```diff
@@ -0,0 +1,3 @@
+export default function printMe() {
+  console.log('I get called from print.js!');
```

Comment:
- [ ] **Required:** Kindly remove this code if it doesn't do anything as well as the `console.log` as this can expose your logic when pushed.